app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,93 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.job_id
+
+    send_spotify_request
+
+    if get_spotify_tracks
+      job_status.update(job_attributes_success)
+    else
+      job_status.update(job_attributes_failure)
+    end
+    job_status.done = true
```

Comment:
`done` is going to be set regardless here, does it make sense to provide it in both `job_attribute_` methods?